From 5506e40196ce11660b39ea7353b98dc0f3e6e6f4 Mon Sep 17 00:00:00 2001
From: Jon TURNEY <jon.turney@dronecode.org.uk>
Date: Thu, 9 Sep 2010 13:43:11 +0100
Subject: [PATCH] Have __glImageSize handle format GL_DEPTH_STENCIL_NV the same way as the server does

There is a mismatch between the way the X server and GLX library
calculates the image size for format GL_DEPTH_STENCIL(|_NV|_EXT)

Signed-off-by: Jon TURNEY <jon.turney@dronecode.org.uk>
---
 src/glx/compsize.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/glx/compsize.c b/src/glx/compsize.c
index 5ba6dc9..ccbacdc 100644
--- a/src/glx/compsize.c
+++ b/src/glx/compsize.c
@@ -74,6 +74,7 @@ __glElementsPerGroup(GLenum format, GLenum type)
    case GL_422_REV_EXT:
    case GL_422_AVERAGE_EXT:
    case GL_422_REV_AVERAGE_EXT:
+   case GL_DEPTH_STENCIL_NV:
    case GL_YCBCR_422_APPLE:
    case GL_LUMINANCE_ALPHA:
       return 2;
-- 
1.7.1

